web/src/app/app.component.ts

```diff
@@ -1,16 +1,38 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, NgZone } from '@angular/core';
+import { Router } from '@angular/router';
 import { environment } from '../environments/environment';
+import { AngularFireAuth } from '@angular/fire/auth';
+import { AppStateService } from './services/app-state.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   public title = 'TribeDashboard';
   public version: string;
 
-  constructor() {
+  constructor(
+    public router: Router,
+    public afAuth: AngularFireAuth,
+    public zone: NgZone,
+    public appState: AppStateService,
+  ) {
     this.version = environment.version;
+    this.router = router;
+  }
+
+  ngOnInit() {
+    this.afAuth.auth.onAuthStateChanged((user) => {
+      if (user) {
+        this.zone.run(() => {
```

Comment:
Why is `zone` needed?

Reply:
Hi Eddie, This is an existing issue in angular where when we redirect to a different webpage outside of angular the zone detection fails. Please refer the following issue link
https://github.com/angular/angular/issues/24168

Reply:
I have never needed to use `zone`. That issue looks closed. 

Leave this to last and I can try to take a look